content/Reducing_Power_Consumption.md

```diff
@@ -59,14 +59,21 @@ power standby mode. For example, a program may use this feature when it
 hasn't sensed keyboard input for a longer period (for example, when
 somebody forgot to turn off the Game Boy).
 
-Before invoking STOP, it might be required to disable Sound and Video
-manually (as well as IR-link port in CGB). Much like HALT, the STOP
-state is terminated by interrupt events. STOP is commonly terminated
-with a joypad interrupt.  A bug may keep STOP from terminating if STOP
-was invoked while a button was held.
-
-During STOP mode, the display will turn white, so avoid using it in your
-game's main loop.
+No licensed rom makes use of STOP outside of CGB speed switching.
+Special care need to be taken if you want to make use of the STOP
+instruction.
+
+On a DMG, disabling the LCD before invoking STOP, leaves the LCD enabled
+drawing a horizontal black line on the screen and damaging the hardware.
+
+On CGB, leaving the LCD enabled when invoking STOP will return in a
+black screen. Except if the LCD is in Mode3, where it will keep drawing
+the current screen.
+
+STOP is terminated by one of the P10 to P13 lines going low. For this 
```

Comment:
Referring to joypad input as "lines" is probably confusing if you don't know the terminology prior, imo they should be changed to more common terms.

Further, behavior resulting from writing $00 should probably made explicit.

Finally, wasn't there an edge case, where the state would only be terminated on a falling *edge*, so any buttons already pressed would not exit STOP immediately? I also remember @LIJI32 talking about STOP potentially locking up in a certain configuration, but I may be misremembering.

Reply:
I initially had something about bugged stop in there, but because I wasn't a 100% sure, it was removed an #110 was opened. So we can use that issue to add that information.

Reply:
About the "lines" that's my EE background getting in there. The P1 part calls it inputs, which would work better and be more consistent indeed.

Reply:
To reduce the cognitive load, and since the relevant input lines directly map to `rP1` bits, I would talk about those, unless that assumption is wrong.